Jenkinsfile

```diff
@@ -337,6 +337,7 @@ pipeline {
                     steps {
                         checkPatch user: GITHUB_USER_USR,
                                    password: GITHUB_USER_PSW,
+                                   branch: 'amd/no-pylint',
```

Comment:
This shouldn't land, correct?  Will -1 just to ensure it doesn't happen accidentally.

Reply:
Removed.